scripts/sign_encrypt.py

```diff
@@ -43,13 +47,13 @@ def get_args(logger):
         ' for offline\n' +
         '                 signing. Takes arguments --uuid, --ta-version,' +
         ' --in, --key,\n'
-        '                 --enc-key (optional) and --dig.\n' +
+        '                 --enc-key (optional), --algo and --dig.\n' +
```

Comment:
By the way, maybe also tag `(optional)` to `--algo` arg in this help message.

Reply:
Fix it